### PR TITLE
[doc] Update field.md (Fields advanced)

### PR DESCRIPTION
**Pull request by vai9er**
Issue: #

### CODE REVIEW

File: docs/lang/articles/basic/field.md [hunk 1]
```diff
@@ -48,7 +44,7 @@ The simplest way to declare a scalar field is to call `ti.field(dtype, shape)`,
   ```python
   f_1d = ti.field(ti.i32, shape=9)  # A 1D field of length 9
   ```
-
+  Over here we are defining a
```
Comment (Olinaaaloompa) [inline docs/lang/articles/basic/field.md:47]: Unfinished sentence?

**Review by Olinaaaloompa — CHANGES_REQUESTED** [review]
I'm now a bit confused about the capitalization thing. Let's discuss it tmr.

File: docs/lang/articles/basic/field.md [hunk 2]
```diff
@@ -331,11 +333,11 @@ The following code snippet declares a tensor field:
 tensor_field = ti.Matrix.field(n=3, m=2, dtype=ti.f32, shape=(300, 400, 500))
 ```
 
-### Access elements in a matrix field
+### Accessing Elements in a Matrix Field
 
 Accessing a matrix field is similar to accessing a vector field: You use an index operator `[]` for field indexing and a second index operator `[]` for matrix indexing.
 
-- To access the `i, j` element of the matrix field `tensor_field`:
+- To access the `ith, jth` element of the matrix field `tensor_field`:
```
Comment (Olinaaaloompa) [inline docs/lang/articles/basic/field.md:340]:
```suggestion
- To access the `i-th, j-th` element of the matrix field `tensor_field`:
```

File: docs/lang/articles/basic/field.md [hunk 3]
```diff
@@ -62,21 +61,25 @@ The simplest way to declare a scalar field is to call `ti.field(dtype, shape)`,
   There is little difference between a 0D field and a 1D field of length 1 except for their indexing rules. You *must* use `None` as the index to access a 0D field and `0` as the index to access a 1D field of length 1:
 
     ```python
-    # f1 and f2 are basically interchangeable
+<<<<<<< HEAD
+
+=======
+
+>>>>>>> da7bac2b5be484ec2648fe058e191497f5803384
```
Comment (neozhaoliang) [inline docs/lang/articles/basic/field.md:68]: A conflict here?

**Review by Olinaaaloompa — APPROVED** [review]
LGTM!

**Pull request merged**